dev_guide/builds.adoc

```diff
@@ -517,36 +521,33 @@ following format:
 
 You can define multiple Docker registry entries in this file. Alternatively, you
 can also add authentication entries to this file by running the `docker login`
-command. The file will be created if it does not exist.
-
-Kubernetes provides the https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/design/secrets.md[Secret]
-resource, which is used to store your configuration and passwords.
-
-To create the `*Secret*` resource from your local `.dockercfg` file, you can run
-following command:
+command. The file will be created if it does not exist. Kubernetes provides
+https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/design/secrets.md[*secrets*],
+which are used to store your configuration and passwords.
 
+. Create the `*secret*` from your local `.dockercfg` file:
++
 ====
 ----
 $ oc secrets new dockerhub ~/.dockercfg
 ----
 ====
++
+This generates a JSON specification of the *secret* named 'dockerhub' and
```

Comment:
Verify the markup for "dockerhub" here. Would suggest "secret" be

```
`*secret`
```

and dockerhub be

```
*dockerhub*
```